src/forklift/lift.py

```diff
@@ -119,6 +119,40 @@ def process_pallets(pallets, is_post_copy=False):
                     log.error('error shipping pallet: %s for pallet: %r', e.message, pallet, exc_info=True)
 
 
+def update_static_for(pallets, config_copy_destinations, force):
+    '''
+    pallets: Pallet[]
+    config_copy_destinations: String[]
+    force: Boolean
+
+    Loop over pallets and check to see if data defined in `static_data` is in `copyDestinations`.
+    If it's not their the data is copied. If it is there and force is True, then the services are
+    shut down and the data is overwritten.
+    '''
+
+    results = ''
+    for pallet in pallets:
+        log.info('checking %s pallet', pallet)
+        for source in pallet.static_data:
+            if not path.exists(source):
+                log.error('static_data: %s does not exist!', source)
+                continue
+
+            destinations = map(lambda d: path.join(d, source.split('\\')[-1]), config_copy_destinations)
+            if all([not path.exists(d) for d in destinations]):
```

Comment:
partial updates of static data aren't possible with `all` correct? It's delete all the static data or none? I would assume we would be using `force` more often than not then? Was there a reason to not allow updating of the missing parts?